packages/utils/src/bufferUtils.ts

```diff
@@ -0,0 +1,11 @@
+export const reverseBuffer = (src: Buffer) => {
```

Comment:
please follow function-per-file pattern, this file should be called `reverseBuffer.ts`

Reply:
let's not be that strict with this, makes sense to have multiple functions in bufferUtils.ts together

Reply:
this pattern is not followed anyway :) see `versionUtils` or `xxsFilters`

second thing is that at some point whole `bufferUtils` from `utxo-lib` should be moved there, right now im only moving essential part of it

Reply:
It will be a mess with these two patterns combined but no big deal